client/my-sites/controller.js

```diff
@@ -112,7 +113,7 @@ function renderNoVisibleSites( context ) {
 	const { getState } = getStore( context );
 	const currentUser = getCurrentUser( getState() );
 	const hiddenSites = currentUser && currentUser.site_count - currentUser.visible_site_count;
-	const signup_url = config( 'signup_url' );
+	const onboardingUrl = getOnboardingUrl( getState() );
```

Comment:
Nit: Could we call `getState()` just once in this function? Perhaps if we store it in a var above?